wazuh/wazuh-dashboard-plugins
# Handle wazuh api error on check updates

### Description
Improvement in error handling from the Wazuh API when retrieving available updates.

### Evidence
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/103193307/2c978a2f-cca1-4acd-85e9-98b21e501500)

### Test
1. Mock a wazuh api error on imposter in the endpoint `/manager/version/check`:
```
{
  "title": "Wazuh Internal Error",
  "detail": "Error in CTI service request: Unauthorized",
  "dapi_errors": {
    "master-node": {
      "error": "Error in CTI service request: Unauthorized",
      "logfile": "WAZUH_HOME/logs/api.log"
    }
  },
  "error": 2100
}
```
3. Go to the API configuration page
4. Click on "Check updates" button
5. Go to the tooltip on the imposter
6. **To ensure proper functionality, test with a real manager and a CTI service**

### Check List
